app/controllers/users_controller.rb

```diff
@@ -29,15 +30,34 @@ def update
   def create
     @user = User.new(user_params)
     if @user.save
+      session[:user_id] = @user.id
       flash[:notice] = "Welcome to the Meta Blog #{@user.username}"
       redirect_to articles_path
     else
       render 'new'
     end
   end
 
+  def destroy
+    @user.destroy
```

Comment:
Whats the difference between `.destroy` & `.delete`?

Reply:
`.destroy` can run callbacks on the model before running the `DELETE` query from the db while `.delete` doesn't. `Destroy` can be used if our models have association or relationships with other models, where we can also destroy associated data when calling destroy on an object with the help of helpers like `:dependent => :destroy_all` before it gets deleted.